modules/stochastic_tools/test/tests/surrogates/libtorch_nn/retrain.i

```diff
@@ -38,6 +38,7 @@
     filename = mynet.pt
     read_from_file = false
     print_epoch_loss = 10
+    max_processes = 1
```

Comment:
Why are you adding this?

Reply:
Right, this is the default value, nice catch!